File: 프로그래머스/unrated/120812. 최빈값 구하기/최빈값 구하기.py
def solution(array):
    answer = 0
    maxN = 0
    for n in array:
        if array.count(n) > maxN:
            maxN = array.count(n)
            answer = n
        elif array.count(n) == maxN and answer != n:
            answer = -1
    return answer

File: 프로그래머스/unrated/120812. 최빈값 구하기/최빈값 구하기.java
class Solution {
    public int solution(int[] array) {
        int answer = 0,max = 0, len = array.length;
        int[] count = new int[len];
        
        for(int i=0;i<len;i++){
            count[i] = 0;
            for(int j=0;j<len;j++){
                if(array[i] == array[j]){
                    count[i] += 1;
                }
            }
            if(max < count[i]){
                max = count[i];
                answer = array[i];
            }
            else if(max == count[i] && answer != array[i]){
                answer = -1;
            }
        }
        return answer;
    }
}